prefect_aws/ecs.py

```diff
@@ -608,15 +608,16 @@ def prepare_for_flow_run(
 
         # Update 'ECSTask' using flow-run context & details
         # update 'env'
-        env_update = flow_run.context.get("env", {})
-        for k, v in env_update.items():
-            env_update[str(k).upper()] = str(v)
+        user_specified_env = flow_run.context.get("env", {})
+        updated_env = {}
+        for k, v in user_specified_env.items():
+            updated_env[str(k).upper()] = str(v)
 
         _job_id_key = ORCHESTRATION_PREFIX + "JOB_ID"
-        if _job_id_key not in env_update:
-            env_update[_job_id_key] = str(flow_run.id)
+        if _job_id_key not in updated_env:
+            updated_env[_job_id_key] = str(flow_run.id)
 
-        new.env.update(env_update)
+        new.env.update(updated_env)
```

Comment:
Updating `env`(new.env) with `updated_env`?
The intention behind `env_update` was to make it seem like an additional update over `env` (new.env).

Or maybe its just diff povs :|

Reply:
Oh got it. Let me rename